src/index.scss

```diff
@@ -55,68 +99,134 @@ html, body, #root, #app, .full-height{
   font-weight: 500;
   
   color: $blue;
-}
-.navigation ul{
-  list-style-type: none;
-  float: right;
 
-  margin: 0;
-  padding: 0;
-}
-.navigation ul li{
-  padding: 25px 25px;
-  text-align: center;
-  text-decoration: none;
-}
-.navigation ul li a{
-  color: $blue;
+  ul {
+    list-style-type: none;
+    float: right;
+  
+    margin: 0;
+    padding: 0;
+
+    li {
+      padding: 25px 25px;
+      text-align: center;
+      text-decoration: none;
+
+      a {
+        color: $blue;
+      
+        border-bottom-style: solid;
+        border-color: transparentize($blue, 1);
+        transition: border-color 0.2s;
+
+        &:hover {
+          border-color: transparentize($blue, 0);
+          transition: border-color 0.2s;
+        }
+      }
+    }
+  }
 
-  border-bottom-style: solid;
-  border-color: transparentize($blue, 1);
-  transition: border-color 0.2s;
-}
-.navigation ul li a:hover{
-  border-color: transparentize($blue, 0);
-  transition: border-color 0.2s;
+  @include tablet {
+    font-size: 12px;
+    font-weight: 400;
+
+    color: $white;
+    background-color: $blue;
+
+    ul li {
+      padding: 25px 15px;
+      a {
+        color: $white;
+        border-bottom-style: solid;
+        border-color: transparentize($white, 1);
+
+        &:hover {
+          border-color: transparentize($white, 1);
+        }
+        &:active {
+          border-color: transparentize($white, 0);
+        }
+      }
+    }
+  }
+
+  @include mobile {
+    /*no scaling for very short devices,
+    scaling matching the mlh badge scaling for avg devices,
+    high scaling for very tall devices
+    */
+    margin-bottom: unquote("max(0px, 10vh - 40px, 0px, 30vh - 190px)");
+  }
 }
 
-.section{
+.section {
   width: 100%;
   max-width: 1100px;
   text-align: center;
   margin: auto;
+
+  @include tablet {
+    flex-direction: column;
+  }
 }
-.left-half, .right-half{
+.right-half {
   flex: 50%;
 }
-.left-half{
-  margin-bottom: 2.5vh;
+.left-half {
+    @extend .right-half;
```

Comment:
Originally, left-half was like flex: 40% and right-half had flex: 60% so the hero-img was larger but I decided that it was unnecessary. They're meant to be imbalanced if they're not both at 50% though